contracts/JBController.sol

```diff
@@ -916,4 +977,20 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
 
     return _fundingCycle.configuration;
   }
+
+  modifier validRates(JBFundingCycleMetadata calldata _metadata) {
```

Comment:
nice

Reply:
I like the gist of the idea but does it make sense to increase contract size by inlining three functions with this modifier when we can just throw these checks in the _configure private function that they all call anyways?

Reply:
true. Although the checks should come before calling to createProjects imo. Validation before any external fns.

Reply:
IMO it would be fine to move these into _configure, since that is the only thing this is validating for, right?

Reply:
Did this, lmk if you want it changed back

Reply:
perfect, im on board